src/services/destination/cdkV2Integration.ts

```diff
@@ -126,7 +127,7 @@ export default class CDKV2DestinationService implements IntegrationDestinationSe
           metaTo.metadata = destInputArray[0].metadata;
           try {
             const doRouterTransformationResponse: RouterTransformationResponse[] =
-              await processCdkV2Workflow(destinationType, destInputArray, tags.FEATURES.ROUTER);
+              await processCdkV2Workflow(destinationType, destInputArray, tags.FEATURES.ROUTER, requestMetadata);
```

Comment:
Did we test other CDKv2 destinations with this contract change ?

Reply:
yes, this contract change is required for new status codes and default value for this field is already added. testing is already done